src/services/expenses.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+let expenses = [];
+
+function init() {
+  expenses = [];
+}
+
+function getAll(userId, category, from, to) {
+  if (userId) {
+    expenses = [expenses.find(expense => expense.userId === +userId)];
```

Comment:
Maybe better to use also `filter` here, because user can have several expenses.

Reply:
agree